### ISSUE TITLE
Setting groups based on SSO

### ISSUE THREAD

**Issue by gueldenstone** [loomio/loomio#9892]
This is more of a question, because I could not find any documentation on it:

Is it possible to set group membership based on data from the SSO solution (OIDC in my case)?

**Comment by robguthrie** [comment]
No, but we are working with a partner to develop an API so that you can add/remove users from groups, discussions, polls with an API key.

SSO for auth, then an API to modify user memberships.

**Comment by poVoq** [comment]
OIDC (Oauth2) does have a groups scope though. So basically there would need to be some matching between the external and the internal group name.

**Comment by gueldenstone** [comment]
Hi sorry for nagging, but are there any plans to integrate this feature?

**Comment by robguthrie** [comment]
We're happy to discuss sponsoring the work, but it won't be prioritized until a customer wants it.
